frontend/src/component/common/StrategySeparator/LegacyStrategySeparator.tsx

```diff
@@ -29,6 +29,9 @@ const StyledCenteredContent = styled(StyledContent)(({ theme }) => ({
     padding: theme.spacing(0.75, 1.5),
 }));
 
+/**
+ * @deprecated remove with 'flagOverviewRedesign' flag. This pollutes a lot of places in the codebase 😞
+ */
```

Comment:
But you're removing the flag now? Why did you add the deprecation notice here? What does it mean?

Reply:
This is not breaking tests, and it's not a case of "just remove this file". I'll split this into a separate PR.